packages/react-icons/generateRollup.js

```diff
@@ -0,0 +1,59 @@
+// @ts-check
+const path = require('path');
+// https://github.com/jprichardson/node-fs-extra/blob/master/docs
+const fs = require('fs-extra');
+// https://www.npmjs.com/package/glob
+const glob = require('glob');
+
+const outDir = path.join(__dirname, '/lib');
+// NOTE: esm comes first to increase chances of it being suggested first in auto-imports
+const outTypes = ['esm', 'cjs'];
+
+// // copy test output to fake react-icons package
+// // (skip this step in real version since output will already be present)
+// fs.rmdirSync(outDir, { recursive: true });
+// for (const outType of outTypes) {
+//   fs.copySync(
+//     path.join(__dirname, `tests/lib/${outType}/3-components-default-named-current`),
+//     path.join(outDir, outType)
+//   );
+// }
+
+const rootInPackage = '@fluentui/react-icons/lib';
+// make a top-level module declaration
+const modules = [
+  `declare module '@fluentui/react-icons' {
+  export * from '${rootInPackage}/esm/index';
+}`,
+];
+
+// make a module declaration for each .d.ts file (and delete the original)
+for (const outType of outTypes) {
+  // This gives a list of relative paths like 'esm/components/Accessibility16Filled.d.ts'
+  const files = glob.sync(`${outType}/**/*.d.ts`, { cwd: outDir });
+  console.log(files);
```

Comment:
Do we still need this console.log? In general it will have a complete set of the files, so won't usually be useful but will still take up space and time logging things.

Reply:
Good point, will remove that!